src/configs/index.js

```diff
@@ -0,0 +1,54 @@
+import ImportContactsIcon from '@material-ui/icons/ImportContacts';
+
+const languages = ['uk', 'en'];
+
+const routes = {
+  pathToNews: '/',
+  pathToNewsDetails: '/news/:id',
+  pathToAddNews: '/newsadd'
+};
+
+const theme = {
+  DARK_THEME: 'dark',
+  LIGHT_THEME: 'light'
+};
+
+const buttonTitles = {
+  DELETE_TITLE: 'Видалити',
+  EDIT_TITLE: 'Редагувати',
+  CREATE_NEWS_TITLE: 'Додати новину'
+};
+
+const statuses = {
+  SUCCESS_ADD_STATUS: 'Успішно додано!'
+};
+
+export const config = {
+  app: {
+    title: 'Horondi Admin Portal',
+    menuCategories: [['News', routes.pathToNews, ImportContactsIcon]],
+    routes,
+    theme,
+    languages,
+    buttonTitles,
+    statuses,
+    serverUrl: 'http://localhost:5000/',
+    drawerWidth: 220,
+    snackBarDuration: 4000,
+    rowsPerPageOptions: [10, 25, 50, 100]
+  },
+  tableHeadRowTitles: {
+    news: ['Аватар', 'Автор', 'Заголовок', 'Дії']
+  },
+  tableSizes: {
+    SMALL_SIZE: 'small',
+    DEFAULT_SIZE: 'medium'
+  },
+  iconSizes: {
+    SMALL_SIZE: 'small',
+    DEFAULT_SIZE: 'default'
+  }
+};
+
+export const IMG_URL =
```

Comment:
Why do we hardcode this link? Can we load this image from azure?

Reply:
Yes, we can, but we don't need `IMG_URL` yet! So, I delete this link.